components/dialog/stories/dialog.mdx

```diff
@@ -0,0 +1,64 @@
+import { ArgTypes, Canvas, Meta, Description, Title } from '@storybook/blocks';
+
+import * as DialogStories from './dialog.stories';
+
+<Meta of={DialogStories} title="Docs" />
+
+<Title of={DialogStories} />
+<Description of={DialogStories} />
+
+## Variants
+
+### Small
+
+<Canvas of={DialogStories.Small} />
+
+### Medium (default)
+
+<Canvas of={DialogStories.Default} />
+
+### Large
+
+<Canvas of={DialogStories.Large} />
+
+### Dismissible
+
+A dialog that can be dismissed without taking an actions. Dismissible dialogs should never have buttons.
+
+<Canvas of={DialogStories.Dismissible} />
+
+### No divider
+
+Dialogs can forgo the divider if they have content that spans the entire width of the dialog.
+
+<Canvas of={DialogStories.NoDivider} />
+
+### Hero
+
+Dialogs can have a hero header.
+
+<Canvas of={DialogStories.WithHero} />
+
+### Scrolling
+
+A dialog without `.spectrum-Dialog--alert` can expand to a larger size to house larger contents.
+
+<Canvas of={DialogStories.WithScroll} />
+
+### Fullscreen
+
+A fullscreen dialog will automatically fill almost all of the available screen space.
```

Comment:
```suggestion
A fullscreen dialog will automatically fill almost all of the available screen space. A margin is included around the outside of the dialog.
```
Some clarification I just discovered and documented while working on Modal.

Reply:
Brilliant! I fixed that with the rest of the mdx removal: https://github.com/adobe/spectrum-css/pull/2833/commits/2cb52cbd0a6373d5963c13d96b8ac856b5cda1fa